uploadNeuroDB/minc_insertion.pl

```diff
@@ -405,7 +405,8 @@
 
 if($acquisitionProtocol =~ /unknown/) {
    $message = "\n  --> The minc file cannot be registered ".
-		"since the AcquisitionProtocol is unknown \n";
+		"since the AcquisitionProtocol is unknown ".
+                "or it did not pass the extra_file_checks\n";
```

Comment:
@gluneau 
If I remember correctly the error you were getting (acquisition protocol does not exist in prod file), I believe this extra message should have been appended to the message on line 432